test/unit/int_algolia/scripts/algolia/helper/jobHelper.test.js

```diff
@@ -2,6 +2,7 @@ const MasterProductMock = require("../../../../../mocks/dw/catalog/MasterProduct
 const VariantMock = require("../../../../../mocks/dw/catalog/Variant");
 const collectionHelper = require("../../../../../mocks/helpers/collectionHelper");
 const jobHelper = require("../../../../../../cartridges/int_algolia/cartridge/scripts/algolia/helper/jobHelper");
+const PromotionMgrMock = require('../../../../../mocks/dw/campaign/PromotionMgr'); // eslint-disable-line no-unused-vars
```

Comment:
You shouldn't need to require something unused.
Jest mocks setup should be declared/loaded in `jest.setup.js`. You can move the content of `PromotionMgr.js` there and remove those lines.